pandas/tests/io/json/test_pandas.py

```diff
@@ -42,20 +41,10 @@ def assert_json_roundtrip_equal(result, expected, orient):
 
 @pytest.mark.filterwarnings("ignore:the 'numpy' keyword is deprecated:FutureWarning")
 class TestPandasContainer:
-    @pytest.fixture(scope="function", autouse=True)
-    def setup(self, datapath):
-        self.dirpath = datapath("io", "json", "data")
+    @pytest.fixture(autouse=True)
+    def setup(self, datapath, monkeypatch):
+        monkeypatch.chdir(datapath("io", "json", "data"))
```

Comment:
why is there even a setup left? this is very messy

Reply:
I can't get rid of the setup just yet because would still need to take care of all of the frame fixtures (this PR only has done Series)

Reply:
But yea goal would be to get rid of it